projects/Mallard/src/App.tsx

```diff
@@ -40,7 +35,9 @@ pushNotifcationRegistration()
 clearOldIssues()
 fetchCacheClear().then((weOk: boolean) => {
     if (weOk) {
-        downloadTodaysIssue()
+        console.log('WE ARE OKAY')
+        //TODO: AW: download todays issue
+        // downloadTodaysIssue()
```

Comment:
Assume this will be put back before this is merged?

Reply:
It need's an `IssueSummary` to make it work, so I'll need to rewrite- so it'll be in the next PR.

Reply:
@jimhunty when does this download attempt happen? On each start?

Reply:
Yes - it attempts to download the days edition. If its already downloaded, it skips.